scripts/scil_dwi_prepare_eddy_command.py

```diff
@@ -15,19 +15,23 @@
 import os
 import subprocess
 
-from dipy.io.gradients import read_bvals_bvecs
 import numpy as np
+
+from dipy.io.gradients import read_bvals_bvecs
```

Comment:
Why did you move that? If I am not mistaken, it should be with the libraries like numpy.

Reply:
@elyz081 this has not been addressed.